src/build.py

```diff
@@ -1646,12 +1648,24 @@ def TestEmwasm():
 def ExecuteEmscriptenTestSuite(name, tests, config, outdir, warn_only=False):
     buildbot.Step('Execute emscripten testsuite (%s)' % name)
     Mkdir(outdir)
+
+    # Before we can run the tests we prepare the installed emscripten
```

Comment:
Is `npm ci` needed before we run *any* tests (i.e. before we use emcc at all), e.g. the torture tests? Or just for the emscripten test suite?

Reply:
Yes, you are right.  In theory it is.   In practice I think we have been getting away with due to the fact that the torture tests are not using the part of emscripten that end up depending on the npm modules (i.e. closure compiler).  At least that is my guess. 

I guess we should really move this to some kind of `PrepareEmscripten` phase?  That happens after archive creation but before any tests run.   I'll leave that for a separate PR though I think